app/src/main/java/com/ataulm/whatsnext/film/RatingsDistributionWidget.kt

```diff
@@ -0,0 +1,137 @@
+package com.ataulm.whatsnext.film
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Rect
+import android.os.Bundle
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import androidx.core.view.ViewCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import androidx.customview.widget.ExploreByTouchHelper
+import com.ataulm.support.resolveColorAttribute
+import com.ataulm.whatsnext.FilmStats.RatingsHistogramBar
+import com.ataulm.whatsnext.R
+import kotlin.math.min
+import kotlin.math.roundToInt
+
+class RatingsDistributionWidget(context: Context, attrs: AttributeSet) : View(context, attrs) {
+
+    private val accessHelper = AccessHelper()
```

Comment:
See this blog post https://medium.com/google-developer-experts/making-custom-views-on-android-accessible-76b8d577be53